import-scripts/subset-impact-data.sh

```diff
@@ -103,9 +103,9 @@ if [ $STUDY_ID == "genie" ]; then
 
         # add age at seq report using cvr/seq_date.txt
         echo "Adding AGE_AT_SEQ_REPORT to $OUTPUT_DIRECTORY/data_clinical_supp_sample.txt"
-        $PYTHON_BINARY $PORTAL_SCRIPTS_DIRECTORY/add-age-at-seq-report.py --clinical-file="$OUTPUT_DIRECTORY/data_clinical_supp_sample.txt" --seq-date-file="$INPUT_DIRECTORY/cvr/seq_date.txt" --age-file="$INPUT_DIRECTORY/ddp/ddp_age.txt" --convert-to-days="true"
+        $PYTHON_BINARY $PORTAL_SCRIPTS_DIRECTORY/add-age-at-seq-report.py --clinical-file="$OUTPUT_DIRECTORY/data_clinical_supp_sample.txt" --sample-file="$INPUT_DIRECTORY/data_clinical_sample.txt" --convert-to-days="true"
```

Comment:
if you define a symbolic variable for the sample file path, you could use it here and also on line 108. That would help avoid a typo difference between the two lines or a maintenance mistake later (changing this line and forgetting to propagate it to 108). It also looks like such a variable could be used in other places in this script.